common/PacketTypes.acn

```diff
@@ -0,0 +1,28 @@
+PacketTypes DEFINITIONS ::= BEGIN
```

Comment:
missing license comment. important: I've fixed template in `master`

Reply:
Fixed

Reply:
Fixed in later commit